constants/map_object_constants.asm

```diff
@@ -31,13 +31,13 @@ DEF OBJECT_MOVEMENT_BYTE_INDEX rb ; 1b
 DEF OBJECT_1C                  rb ; 1c
```

Comment:
```suggestion
DEF OBJECT_ANON_JUMPTABLE_INDEX                  rb ; 1c
```
You'll have to fix the formatting and the associated wram label, and everywhere it's used.

Reply:
(If you agree with this, I can push the required changes to the PR for you. Since I know you are busy.)